angular_hover/src/app/emp-form/emp-form.component.ts

```diff
@@ -0,0 +1,65 @@
+import { CommonModule } from '@angular/common';
+import { Component, NgModule} from '@angular/core';
+import { FormControl, FormGroup, FormsModule, ReactiveFormsModule, Validators} from '@angular/forms';
+import { IemployeeForm } from '../employee-form.interface';
+import { ProjectPipePipe } from '../project-pipe.pipe';
+import { NgSelectComponent } from '@ng-select/ng-select';
+import { EmpHoverDirective } from '../emp-hover.directive';
+
+@Component({
+  selector: 'app-emp-form',
+  standalone: true,
+  imports: [CommonModule,ReactiveFormsModule,ProjectPipePipe,FormsModule,CommonModule,NgSelectComponent,EmpHoverDirective],
+  templateUrl: './emp-form.component.html',
+  styleUrl: './emp-form.component.scss'
+})
+
+export class EmpFormComponent
+{
+  formSubmitted :boolean= false;
+  employeeDetails:IemployeeForm|object=null;
```

Comment:
IEmployeeForm, why use object?

Reply:
it at first showed error when given only that, said its object is being stored into employeeDetails